### PR TITLE
[Merged by Bors] - doc: Better explain `initialize_simps_projections`

### PR DESCRIPTION
**Pull request by YaelDillies**
The current docstring mentions `simpsAttr` in the very first sentence, but not `simps`. This striked me as odd as the former is an implementation detail while the latter is the actual useful interface.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Tactic/Simps/Basic.lean [hunk 1]
```diff
@@ -269,31 +269,36 @@ syntax simpsRule := simpsRule.prefix <|> simpsRule.rename <|> simpsRule.erase <|
...
+This default behavior is customisable as such:
 * You can disable a projection by default by running
   `initialize_simps_projections Equiv (-invFun)`
   This will ensure that no simp lemmas are generated for this projection,
-  unless this projection is explicitly specified by the user.
+  unless this projection is explicitly specified by the user (as in
+  `@[simps invFun] def myEquiv : Equiv _ _ := _`).
 * Conversely, you can enable a projection by default by running
-  `initialize_simps_projections Equiv (+toEquiv)`.
+  `initialize_simps_projections MulEquiv (+toEquiv)`.
```
Comment (b-mehta) [inline Mathlib/Tactic/Simps/Basic.lean:289]: Hmm, can we somehow make the examples more consistent throughout? Switching between MulEquiv and Equiv and myEquiv is somewhat confusing, and somewhat suggests that changing `Equiv` is relevant for setting up `simps` on `MulEquiv`. Perhaps we could use `MulEquiv` as the running example throughout?

File: Mathlib/Tactic/Simps/Basic.lean [hunk 2]
```diff
@@ -269,31 +269,36 @@ syntax simpsRule := simpsRule.prefix <|> simpsRule.rename <|> simpsRule.erase <|
...
+By default, tagging a definition of an element `myObj` of a structure `MyStruct` with `@[simps]`
+generates one `@[simp]` lemma `myObj_myProj` for each projection `myProj` of `MyStruct`. There are a
+few exceptions to this general rule:
 * For algebraic structures, we will automatically use the notation (like `Mul`)
   for the projections if such an instance is available.
 * By default, the projections to parent structures are not default projections,
   but all the data-carrying fields are (including those in parent structures).
+
+This default behavior is customisable as such:
 * You can disable a projection by default by running
```
[the remark this answers is not in the document]
Comment (YaelDillies) [reply Mathlib/Tactic/Simps/Basic.lean:283]: Whoops, accident

File: Mathlib/Tactic/Simps/Basic.lean (hunk 1 above)
Comment (YaelDillies) [reply Mathlib/Tactic/Simps/Basic.lean:289]: What about now?

File: Mathlib/Tactic/Simps/Basic.lean (hunk 2 above)
[the remark this answers is not in the document]
Comment (joneugster) [reply Mathlib/Tactic/Simps/Basic.lean:283]: I assume this is resolved?

**Comment by joneugster** [comment]
LGTM

maintainer merge

**Comment by b-mehta** [comment]
Nice!

bors merge

**Pull request closed without merging**